libexec/rtld-elf/rtld_c18n.h

```diff
@@ -217,8 +259,8 @@ func_sig_legal(struct func_sig sig)
 /*
  * This macro can only be used in a function directly invoked by a trampoline.
  */
-#define	c18n_return_address()						\
-	(C18N_ENABLED ? get_trusted_stk()->pc : __builtin_return_address(0))
+#define	c18n_return_address()	(C18N_ENABLED ?				\
```

Comment:
There should probably be a non-c18n fallback implementation of this so you don't need to ifdef every use.

Reply:
Will do in another PR